src/frontend/Gopkg.lock

```diff
@@ -35,7 +35,7 @@
     "src/frontend/money",
   ]
   pruneopts = "UT"
-  revision = "6d969441585ade8c91c235115c7cdb12ac61354f"
+  revision = "86c8c06cc1e0aa616796a19493f042466d93098d"
```

Comment:
actually this is bad. I'm not sure why the repo itself is ending up at Gopkg.lock.